### DISCUSSION TITLE
How to enable basic authentiacation in Skywalking

### DISCUSSION THREAD

**Discussion by roshnithadevus27** [apache/skywalking#11953] [Q&A]
Hi Team,

Could you please tell us the procedure for enabling the basic authentication in Skywalking?

**Comment by wu-sheng** [comment]
We provided username and password configurations, ref https://skywalking.apache.org/docs/main/latest/en/setup/backend/storages/elasticsearch/#elasticsearch

**Reply by AjinThankachan** [reply]
How can we enable the authentication you enabled on the first page for every access?
![image](https://github.com/apache/skywalking/assets/25667606/0ef40935-2674-4601-a77c-21e3604909d2)

**Reply by Superskyyy** [reply]
@AjinThankachan This is just a simple basic auth where you can get from every gateway like nginx, there's nothing special on SkyWalking side AFAIK https://docs.nginx.com/nginx/admin-guide/security-controls/configuring-http-basic-authentication/
